kolibri/utils/version.py

```diff
@@ -241,7 +241,7 @@ def get_git_describe():
         # repo - it's safe.
         version_string = p.communicate()[0].rstrip()
         return version_string
-    except EnvironmentError:
+    except EnvironmentError, OSError:
```

Comment:
If `OSError` is derived from `EnvironmentError`, shouldn't a `OSError` be caught by `except EnvironmentError`? 
https://docs.python.org/2/library/exceptions.html#exceptions.OSError

Reply:
I added the `EnvrionmentError` to fix a similar issue: https://github.com/learningequality/kolibri/pull/2199#issuecomment-330703166

Reply:
You're right, it should be caught! I think this PR is redundant then.